src/claim_canister/tests/setup.rs

```diff
@@ -0,0 +1,136 @@
+
+    let minting_account = Account { owner: ledger_id, subaccount: None };
+    let owner_account = Account { owner, subaccount: None };
+    let initial_balances = vec![(owner_account, Nat::from(10_000_000_000_000u64))];
+
+    let init_args = InitArgs {
+        minting_account,
+        fee_collector_account: None,
+        transfer_fee: Nat::from(10_000u64),
+        decimals: Some(6),
+        max_memo_length: Some(64),
+        token_symbol: "TEST".to_string(),
+        token_name: "Test Token".to_string(),
+        metadata: vec![],
+        initial_balances,
+        feature_flags: Some(FeatureFlags { icrc2: true }),
+        maximum_number_of_accounts: None,
+        accounts_overflow_trim_quantity: None,
+        archive_options: ArchiveOptions {
+            num_blocks_to_archive: 1000,
+            max_transactions_per_response: Some(100),
+            trigger_threshold: 2000,
+            max_message_size_bytes: Some(1024 * 1024),
+            cycles_for_archive_creation: Some(1_000_000_000_000),
+            node_max_memory_size_bytes: Some(32 * 1024 * 1024),
+            controller_id: ledger_id,
+            more_controller_ids: None,
+        },
+    };
+    let ledger_arg = LedgerArg::Init(init_args);
+    pic.install_canister(ledger_id, ledger_wasm, encode_args((ledger_arg,)).unwrap(), None);
+
+    let claim_wasm = std::fs::read(CLAIM_WASM_PATH)
+        .expect("Build first: cargo build --target wasm32-unknown-unknown --release -p claim_canister");
+    let claim_canister = pic.create_canister();
+    pic.add_cycles(claim_canister, 2_000_000_000_000);
+    pic.install_canister(claim_canister, claim_wasm, encode_args((owner,)).unwrap(), None);
+
+    let pool_wasm = std::fs::read(POOL_WASM_PATH)
+        .expect("Build first: cargo build --target wasm32-unknown-unknown --release -p pool_canister");
+    let pool_canister = pic.create_canister();
+    pic.add_cycles(pool_canister, 2_000_000_000_000);
+    
+    pic.install_canister(
+        pool_canister, 
+        pool_wasm, 
+        encode_args((ledger_id, claim_canister)).unwrap(),
+        None
+    );
+
+    // Transfer funds to pool canister for slashing
+    let transfer_to_pool = TransferArg {
+        from_subaccount: None,
+        to: Account { owner: pool_canister, subaccount: None },
+        amount: Nat::from(1_000_000_000_000u64),
+        fee: Some(Nat::from(10_000u64)),
+        memo: None,
+        created_at_time: None,
+    };
+    
+    pic.update_call(
+        ledger_id,
+        owner,
+        "icrc1_transfer",
+        encode_args((transfer_to_pool,)).unwrap(),
+    ).unwrap();
+
+    // Fund pool with deposit
+    let current_episode_bytes = pic
+        .query_call(pool_canister, owner, "get_current_episode_id", encode_args(()) .unwrap())
+        .unwrap();
+    let mut current_episode: u64 = Decode!(&current_episode_bytes, u64).unwrap();
+    
+    while current_episode % 3 != 2 {
```

Comment:
Can you import the util function from pool_canister instead of repeating yourself?